perun-base/src/main/resources/perun-roles.yml

```diff
@@ -7709,6 +7709,25 @@ perun_policies:
 
 
 # A list of Perun management rules that are loaded to the PerunPoliciesContainer.
+#  primary_object serves to determine with which object is the role primarily connected. Other objects are just complementary.
+#  assign_to_objects is a map of objects which can be assigned with the role. Key is a object name and value is mapping to the database.
+#             Example entry: key: Resource; value: resource_id
+#  assignment_check is a list of maps defining which of the assigned objects should be checked for being critical (requiring MFA) when setting the role.
+#             Example entry: key: MFA; value: Resource <- If resource is critical, MFA is required
+#             Example entry: {} <- No MFA is required to set this role
+#             Example entry: Key: MFA; value: <- No value means MFA is always required to set this role
+#  entities_to_manage is a map of entities which can be set to the role. Key is a entity name and value is mapping to the database.
+#             Example entry: key: User; value: user_id
+#  privileged_roles_to_manage is a list of maps where each map entry consists from a role name as a key and a role object as a value.
```

Comment:
i miss some description of the role here - i think this property means 'who can set this role to another user/group' (so voadmin is privileged to set groupadmin role to user/group etc.), would be nice to update it in roleManagermentRules class too